code/model/submissions/SubmittedForm.php

```diff
@@ -83,6 +83,10 @@ public function getCMSFields() {
 	 * @return boolean
 	 */
 	public function canCreate($member = null) {
+		$extended = parent::canCreate($member);
```

Comment:
This should be `$this->extendedCan('canCreate', $member);` not `parent::canCreate();`

LIkewise for each of the below.